practice/tic_tac_toe/coords_board.py

```diff
@@ -0,0 +1,115 @@
+"""Implements Tic-Tac-Toe board using three-element tuples."""
+
+TOKENS = ['X', 'O']
+
+
+class CoordsTTTBoard:
```

Comment:
Put class docstrings. Even if it's just duplicating the module docstring, but in larger programs the module also contains helper code so it probably has a different docstring. Go through your three classes and address this.